pywt/__init__.py

```diff
@@ -35,5 +35,11 @@
 
 from pywt.version import version as __version__
 
+import numpy as np
+if LooseVersion(np.__version__) >= LooseVersion("1.14"):
```

Comment:
`LooseVersion` doesn't work correctly in some cases (and neither does `StrictVersion`), better to use 

    np.lib.NumpyVersion(np.__version__) < '1.14.0'